Dockerfile.okd

```diff
@@ -0,0 +1,22 @@
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
+WORKDIR /go/src/github.com/operator-framework/operator-marketplace
+COPY . .
+RUN make osbs-build
+
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
+RUN useradd marketplace-operator
+USER marketplace-operator
+COPY --from=builder /go/src/github.com/operator-framework/operator-marketplace/build/_output/bin/marketplace-operator /usr/bin
+ADD manifests /manifests
+ADD defaults /defaults
+RUN sed -i 's;registry.redhat.io;registry.access.redhat.com;' /defaults/03_community_operators.yaml
+
+LABEL io.k8s.display-name="OpenShift Marketplace Operator" \
```

Comment:
Should this have different labels to indicate OKD?

Reply:
No, these would land in different repos and don't get mixed up. Other than that these are identical images